### ISSUE TITLE
Date of publication - Metadata

### ISSUE THREAD

**Issue by meyerch** [geosolutions-it/unesco-ihp#57]
Before, the time of publication was composed of a date + an hour (hh:mm AM/PM).
Now, only the date of the publication appears in the metadata. ‘Midnight’ appears by default every time. 

Please allow for the hour of the publication to be displayed after the date (like it was before) 
For example, it should be: ‘Publication Date: Jan. 29, 2017, 3:46 AM’ and not “Publication Date: Jan. 29, 2017, midnight’

**Comment by meyerch** [comment]
ACCEPTED

**Comment by meyerch** [comment]
I am reopening this one as I noticed something I did not see during the test. 

There is an issue with the AM/PM hour.
Yesterday, I uploaded layers in the afternoon, but the sytem ended up referencing them in the morning. 
ex for this layer: http://ihp-wins-dev.geo-solutions.it/layers/geonode:wash_deaths_dalys2 which I uploaded at 3:42 PM, but the metadata says 3:42 AM.
Same with http://ihp-wins-dev.geo-solutions.it/layers/geonode:precipitation1_1, http://ihp-wins-dev.geo-solutions.it/layers/geonode:wash_deaths_number1, http://ihp-wins-dev.geo-solutions.it/layers/geonode:waterabstraction_mln_m3, etc.
